### ISSUE TITLE
UI for the app/ Building basic pages

### ISSUE THREAD

**Issue by AdityaPrasad275** [AdityaPrasad275/Attendance-verification-app#2]
An high-level overview is developed in [here](https://github.com/AdityaPrasad275/Attendance-verification-app/blob/main/plan/UI_and_registration.md). Let's develop the pages first without any fancy db logic. The following are basic pages for the app -

1. Login/Registration:
   - Entry point for users to log in or register for a new account.
   - Intuitive and secure UI for seamless login or registration process.

2. Dashboard:
   - Overview of attendance statistics and information.
   - Eye-catching visual elements displaying overall attendance stats.
   - Link/button to view enrolled courses (students) or courses taught (instructors).
   - Option to enroll in new courses (students) or create new courses (instructors).

3. Calendar:
   - Integration with Google Calendar to view academic schedule.
   - User-friendly display of calendar events, including classes and important dates.

4. Attendance Marking/Classes Today:
   - Displays classes scheduled for the current day.
   - Each class represented as an object/blob/box, showing relevant details like course name, time, and location.
   - Clicking on a class allows students to mark their attendance.
   - Clear and intuitive UI for easy attendance marking.

5. Reappeal:
   - Enables students to request an appeal for missed attendance.
   - View list of absent classes within a specified timeframe.
   - Initiate appeal process with a reappeal button.
   - Track status of reappeal requests.

Classes in the Attendance Verification App are represented as objects/blobs/boxes. They provide relevant information such as the course name, time, and location. Students can interact with classes by clicking on them to access the attendance marking feature. The UI ensures a clear and intuitive user experience for efficient attendance management.

Please refer to the corresponding plans for detailed implementation guidelines and high-level overviews of each feature. Your contributions and suggestions are highly appreciated as we strive to create an outstanding attendance verification app. Don't forget to communicate your preferred color palette for the UI, including both light mode and dark mode options.

**Comment by Ccivam** [comment]
You havent specified any Database or any backend framework

**Comment by AdityaPrasad275** [comment]
Yeah i have no understanding of backend (Firebase) yet lol. Would greatly appreciate if you could help me in understanding how that would work.

**Comment by Ccivam** [comment]
Even Idont know firebase.I only know NodeJS.But you have freecodecamp
